theme/tutorial-sidebar.less

```diff
@@ -1030,7 +1028,6 @@
                     min-height: @tutorialSimulatorMinHeight;
                 }
                 .simframe {
-                    padding-bottom: unset !important;
```

Comment:
Can you double check arcade looks fine without this? we might need to have this in the non `#root.tutorialSimSidebar` case (I guess in it's own `#root.tabTutorial:not(.fullscreensim):not(.tutorialSimSideBar) .simulator-container:not(.hidden) .simframe {`)

Reply:
I took a look and didn't notice any unexpected behavior. Seems like, on short screens, it's doing the same thing the live site is doing...

Target here if you want to double check: https://arcade.makecode.com/app/a5958ad6297dd96d3b68623bb7e827895f34b074-52a96d1738

(I did notice a different bug, but I think that's related to the resizable instructions change, not this one. Filed that here: https://github.com/microsoft/pxt-arcade/issues/5929)